polygeist/lib/Conversion/PolygeistToLLVM/PolygeistToLLVM.cpp

```diff
@@ -60,6 +60,42 @@ namespace mlir {
 #undef GEN_PASS_DEF_CONVERTPOLYGEISTTOLLVM
 } // namespace mlir
 
+/// Return \p mrTy's address space as an integer (being 0 the default).
+static std::optional<unsigned>
+getMemorySpaceAsInt(const TypeConverter &converter, MemRefType mrTy) {
+  Attribute memorySpace = mrTy.getMemorySpace();
+  if (!memorySpace)
+    return 0;
```

Comment:
Can we use an enumerator rather than 0 to indicate the global address space ?

Reply:
See other response in [PolygeistOps.cpp](https://github.com/intel/llvm/pull/11193/files/639d93306d83e468fddc7fb208f320cc321e2ee8#diff-a38b1bc5f7114486fc3be82035fa07f0289ef37393acde434381fb605b7013fd)